main0.py

```diff
@@ -205,7 +208,14 @@ def do_settings():
     F_LIMIT=int(allsettings['backup_size']*1024*1024)
     if F_LIMIT==0:
         F_LIMIT=9223372036854775807
-
+    PBKEY = allsettings['PB_ApiKey']
+    if pb is None:
```

Comment:
No need to check, if that's None, since it always is. I only check the command line arguments for that, since they are allowed to overwrite settings. Also I'll add another boolean settings to make pushbullet optional.